### PR TITLE
reorder modules

### PR DESCRIPTION
**Pull request by ryanchou-dev**
ty @izhang05 for the suggestion!

---
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: content/ordering.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ const MODULE_ORDERING: { [key in SectionID]: Chapter[] } = {
   bronze: [
     {
       name: 'Getting Started',
-      items: ['time-comp', 'rect-geo', 'intro-ds', 'simulation'],
+      items: ['time-comp', 'intro-ds', 'simulation', 'rect-geo'],
```
Comment (SansPapyrus683) [inline content/ordering.ts:73]: thought simulation should be put in front of intro ds? idk lol

Comment (ryanchou-dev) [reply content/ordering.ts:73]: oh, sure

Comment (SansPapyrus683) [reply content/ordering.ts]: actually uh can you change it back
think ds is more appropriate at the front
really sorry

Comment (ryanchou-dev) [reply content/ordering.ts]: LMAO

**Pull request merged**